client/src/services/base.ts

```diff
@@ -15,6 +15,8 @@ export class BaseService {
 }
 
 export const graphClient = Client.init({
+    // For supporting beta version
+    defaultVersion: 'beta',
```

Comment:
If i comment out this line and try to call

graphClient.api(`/reports/microsoft.graph.getSharePointSiteUsageDetail(period='D${period}')?$format=application/json`).get()

I get this error:

https://prnt.sc/e1WMhXzvFyBd

Since the endpoint is 

https://graph.microsoft.com/beta/reports/microsoft.graph.getSharePointSiteUsageDetail(period='D${period}')?$format=application/json

It is /beta and not /v1, hence with my basic understanding it implies the url by default points to /v1